asr:
  ros__parameters:
    service_provider: "ifly"  # baidu ifly
    baidu_api_key: "yqapHe9IKnKZZdKIkvv0r7YQ"
    baidu_secret_key: "uWXoblwoT1YR5BZPoN3SUrttXfPxDm3I"
    ifly_appid: "6f20e2c2"
    ifly_apikey: "fc5fc33f5d17487a93263120e7c6d393"
    ifly_apisecret: "NzMwZTYwYTY0MWZkNjYyMGM0YjJjMjE4"